src/include/module/audio/format.h

```diff
@@ -0,0 +1,51 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2023 Intel Corporation. All rights reserved.
+ *
+ * Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
+ *	   Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *	   Keyon Jie <yang.jie@linux.intel.com>
+ *	   Adrian Warecki <adrian.warecki@intel.com>
+ */
+
+#ifndef __MODULE_AUDIO_FORMAT_H__
+#define __MODULE_AUDIO_FORMAT_H__
+
+#include <stdint.h>
+#include "../ipc/stream.h"
+
+static inline uint32_t get_sample_bytes(enum sof_ipc_frame fmt)
+{
+	switch (fmt) {
+	case SOF_IPC_FRAME_S16_LE:
+		return 2;
+	case SOF_IPC_FRAME_S24_3LE:
```

Comment:
SOF_IPC_FRAME_S24_4LE?

Reply:
This function was moved as is from [src/include/sof/audio/format.h](https://github.com/thesofproject/sof/pull/8365/files#diff-6d0f97b1dc37dbd4b9302e4c4456638dd19426a8126cc311837e0225b4e7827eL173)